hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieCompactionMetadata.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2016 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.uber.hoodie.io.compact;
+
+import com.uber.hoodie.common.model.HoodieCommitMetadata;
+
+/**
+ * Place holder for the compaction specific meta-data, uses all the details used in a normal HoodieCommitMetadata
+ */
+public class HoodieCompactionMetadata extends HoodieCommitMetadata {
```

Comment:
CommitMetadata needs to be cleaned up carefully, to not affect the current production stuff/ 

But some naming/organization like below 
HoodieTimelineInstantMetadata 
   ->  CommitInstantMetadata (this is new name for HoodieCommitMetadata) 
   ->  CompactionInstantMetadata
   -> CleanerInstantMetadata 

might be a better long term thing..